src/App.jsx

```diff
@@ -1,104 +1,110 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentById(id) {
+  return commentsFromServer.filter(comment => comment.postId === id) || [];
```

Comment:
The `getCommentById` function uses `filter`, which is correct for returning multiple comments associated with a post. Ensure that this function correctly returns an empty array when no comments are found for a given post ID.